src/main/java/net/mcreator/ui/browser/WorkspaceFileBrowser.java

```diff
@@ -91,7 +91,7 @@ public class WorkspaceFileBrowser extends JPanel {
 			g.setColor(new Color(111, 111, 111));
 			g.setFont(getFont().deriveFont(10f));
 			if (getText().trim().equals(""))
-				g.drawString("Search by file name", 2, 17);
+				g.drawString(L10N.t("workspace_file_browser.search"), 2, 17);

```

Comment:
Not sure about this one, maybe it should be removed as well?

Reply:
Removed or added? It seems to be hardcoded so far

Reply:
Should I use the key instead of direct string?

Reply:
I guess so, yep

Reply:
Then it's done already.